DashKit/PodStatusExtension.swift

```diff
@@ -158,6 +155,8 @@ extension PodCommError: LocalizedError {
             
         case .bluetoothUnauthorized:
             return LocalizedString("Bluetooth unauthorized", comment: "Error description for PodCommError.bluetoothUnauthorized")
+        case .systemError(let systemError):
```

Comment:
[nit] No blank line between `case` statements to mirror the rest of the `switch` statement.

Reply:
Thanks, will get this in the next Dash pr.